Makefile.pre.in

```diff
@@ -564,6 +564,14 @@ LIBEXPAT_HEADERS= \
 
 LIBEXPAT_A= Modules/expat/libexpat.a
 
+#
+# SQLite stuff
+#
+
+# For setup.py
+PY_SQLITE_CFLAGS=@PY_SQLITE_CFLAGS@
+PY_SQLITE_LDFLAGS=@PY_SQLITE_LDFLAGS@
```

Comment:
My PR GH-29534 adds some automated variables like ``MODULE_{name}_CFLAGS``. Let me pull out the interesting bits as a separate PR so you can build on top of it.

```
MODULE__SQLITE3_=yes  # yes, missing, disabled, n/a
MODULE__SQLITE3_CFLAGS=
MODULE_SQLITE3_LDFLAGS=
```

Reply:
SGTM!

Reply:
Leaving this PR as draft until you've merged your new PR.